### ISSUE TITLE
IntelliCode Completions does not work on the portable version

### ISSUE THREAD

**Issue by mike2003** [MicrosoftDocs/intellicode#521]
I just tried installing IntelliCode **Completions** on clean fresh Vscode.
_Version: 1.84.2
Date: 2023-11-09T10:51:52.184Z
Electron: 25.9.2 ElectronBuildId: 24603566
Chromium: 114.0.5735.289
Node.js: 18.15.0 V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10_

And in the console I will get an error
```
[Worker thread] Unhandled rejection error: fetch failed
[Worker thread] Unhandled rejection error stack: TypeError: fetch failed
    at Object.fetch (node:internal/deps/undici/undici:11413:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[Worker thread] Unhandled rejection error: fetch failed
[Worker thread] Unhandled rejection error stack: TypeError: fetch failed
    at Object.fetch (node:internal/deps/undici/undici:11413:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
Extension version: 1.0.22 034a5c7
```
I also have a separate version Vscode installed normally (1.78.2 ), and everything works there.
I only found the problem in the **portable** version.
How can I solve the problem? Help is needed

**Comment by mike2003** [comment]
I did a little research and was unable to get IntelliCode Completions to run on the new version of vscode in any way.

**Comment by CleyFaye** [comment]
It is broken on any vscode based on recent node version. You can fiddle a bit and disable fetch support, but in turn this breaks other extensions.
To work correctly this would need to be rebuilt.

**Comment by AaronBeaudoin** [comment]
I am also experiencing this issue, here is my output after typing a few characters in a `.cs` file:

```
Inline Completions document change subscription registered.
Initializing IntelliCode Inline Completions
[Completions][2]: Completions worker with process id 30780 has been created
Inline Completions Initialized
[Completions][1]: [inlineCompletionProvider.provideInlineCompletionItems() 0]  Entered provideInlineCompletionItems() try block
[Completions][1]: [inlineCompletionProvider.provideInlineCompletionItems() 0] Skipping request
[Completions][2]: [CompletionState.getLineCompletion()] : Best suffix
	First token score log confidence: 0 < -0.35667494393873245
	Suggestion: 
[Completions][2]: [inlineCompletionProvider.provideInlineCompletionItems() 0]  Candidate completion: 
[Completions][2]: [inlineCompletionProvider.provideInlineCompletionItems() 0]  Current line with candidate:

[Completions][1]: [inlineCompletionProvider.provideInlineCompletionItems() 0]  Completion was filtered out because Completion line is an empty string
[Completions][2]: [Child process] Worker with pid 30780 has started
[Completions][2]: [Child process] Requested setting state for intellicode-completions.internal.defaultConfidenceThreshold = 0.39 (default is 0.39)
[Completions][2]: [Child process] Requested setting state for intellicode-completions.internal.defaultConfidenceThreshold = 0.39 (default is 0.39)
[Completions][2]: [Child process] Requested setting state for intellicode-completions.internal.defaultConfidenceThreshold = 0.39 (default is 0.39)
[Completions][2]: [Child process] Requested setting state for intellicode-completions.internal.defaultConfidenceThreshold = 0.39 (default is 0.39)
[Completions][2]: [Child process] Requested setting state for intellicode-completions.internal.defaultConfidenceThreshold = 0.39 (default is 0.39)
[Completions][2]: [Child process] Requested setting state for intellicode-completions.internal.defaultConfidenceThreshold = 0.39 (default is 0.39)
[Completions][2]: [Child process] Requested setting state for intellicode-completions.internal.defaultConfidenceThreshold = 0.39 (default is 0.39)
[Completions][4]: [Child process] Unhandled rejection error: fetch failed
[Completions][4]: [Child process] Unhandled rejection error stack: TypeError: fetch failed
    at Object.fetch (node:internal/deps/undici/undici:11413:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[Completions][4]: [Child process] Unhandled rejection error: fetch failed
[Completions][4]: [Child process] Unhandled rejection error stack: TypeError: fetch failed
    at Object.fetch (node:internal/deps/undici/undici:11413:11)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[Completions][1]: [inlineCompletionProvider.provideInlineCompletionItems() 1]  Entered provideInlineCompletionItems() try block
[Completions][1]: [inlineCompletionProvider.provideInlineCompletionItems() 1] Starting request
[Completions][1]: [WorkerLineCompletionService.request() 2]  Entered
[Completions][2]: [inlineCompletionProvider.provideInlineCompletionItems() 1]  Recieved cancellation event, ran for 560.2026000022888
[Completions][1]: [inlineCompletionProvider.provideInlineCompletionItems() 2]  Entered provideInlineCompletionItems() try block
[Completions][1]: [inlineCompletionProvider.provideInlineCompletionItems() 2] Starting request
[Completions][4]: [inlineCompletionProvider.provideInlineCompletionItems() 1]  Caught error when requesting inline completion: Error: Operation took longer than 5000ms.
[Completions][2]: [inlineCompletionProvider.provideInlineCompletionItems() 1] 
	Wait: 0.3776000738143921ms
	Full inference: 5002.360299944878
[Completions][4]: [inlineCompletionProvider.provideInlineCompletionItems() 1] Unexpected exception: Operation took longer than 5000ms.
Error: Operation took longer than 5000ms.
	at Timeout._onTimeout (c:\Users\AaronBeaudoin\.vscode\extensions\ms-dotnettools.vscodeintellicode-csharp-0.1.26-win32-x64\dist\node_modules\@vsintellicode\completions-csharp\index.js:2:66913)
	at listOnTimeout (node:internal/timers:569:17)
	at process.processTimers (node:internal/timers:512:7)
[Completions][1]: [WorkerLineCompletionService.request() 3]  Entered
[Completions][4]: [inlineCompletionProvider.provideInlineCompletionItems() 2]  Caught error when requesting inline completion: Error: Operation took longer than 5000ms.
[Completions][2]: [inlineCompletionProvider.provideInlineCompletionItems() 2] 
	Wait: 4273.758199930191ms
	Full inference: 5001.661500096321
[Completions][4]: [inlineCompletionProvider.provideInlineCompletionItems() 2] Unexpected exception: Operation took longer than 5000ms.
Error: Operation took longer than 5000ms.
	at Timeout._onTimeout (c:\Users\AaronBeaudoin\.vscode\extensions\ms-dotnettools.vscodeintellicode-csharp-0.1.26-win32-x64\dist\node_modules\@vsintellicode\completions-csharp\index.js:2:66913)
	at listOnTimeout (node:internal/timers:569:17)
	at process.processTimers (node:internal/timers:512:7)
```

**Comment by vivlimmsft** [comment]
Duplicate of #491, which was fixed with yesterday's update to v2.0.0.
